fedoracommunity/connectors/bodhiconnector.py

```diff
@@ -541,11 +565,11 @@ def query_active_releases(self, filters=None, **params):
                 release_tag[tag] = r
             else:
                 if 'epel' in tag:
-                    stable_tag = tag
-                    testing_tag = tag + '-testing'
+                    stable_tag = release['stable_tag']
+                    testing_tag = release['testing_tag']
                 else:
-                    stable_tag = tag + '-updates'
-                    testing_tag = stable_tag + '-testing'
+                    stable_tag = release['stable_tag']
+                    testing_tag = release['testing_tag']
```

Comment:
I don't see any difference between the `if` and the `else`, both are executing the same code 
```
stable_tag = release['stable_tag']
testing_tag = release['testing_tag']
```